packages/ember-data/lib/adapters/build-url-mixin.js

```diff
@@ -46,17 +46,17 @@ export default Ember.Mixin.create({
     @param {(String|Array|Object)} id single id or array of ids or query
     @param {(DS.Snapshot|Array)} snapshot single snapshot or array of snapshots
     @param {String} requestType
-    @param {Object} query object of query parameters to send for findQuery requests.
+    @param {Object} query object of query parameters to send for query requests.
     @return {String} url
   */
   buildURL: function(modelName, id, snapshot, requestType, query) {
     switch (requestType) {
-      case 'find':
-        return this.urlForFind(id, modelName, snapshot);
+      case 'findRecord':
+        return this.urlForFindRecord(id, modelName, snapshot);
       case 'findAll':
         return this.urlForFindAll(modelName);
```

Comment:
This breaks backwards compatibility as well right? We should keep the old hooks in for 1.13 no?